--- origsrc/xfce4-taskmanager-0.4.1/src/functions.c	2008-09-15 16:17:08.000000000 -0500
+++ src/xfce4-taskmanager-0.4.1/src/functions.c	2009-02-11 22:45:27.093750000 -0600
@@ -52,7 +52,7 @@ gboolean refresh_task_list(void)
 			if(new_tmp->pid == tmp->pid)
 			{
 /* FIXME */
-#ifdef __linux
+#if defined(__linux) || defined(__CYGWIN__)
 				tmp->old_time = tmp->time;
 				tmp->time = new_tmp->time;
 
@@ -64,7 +64,7 @@ gboolean refresh_task_list(void)
 				    strcmp(tmp->state,new_tmp->state) ||
 				    tmp->vsize != new_tmp->vsize ||
 				    tmp->rss != new_tmp->rss ||
-#ifdef __linux
+#if defined(__linux) || defined(__CYGWIN__)
 				    tmp->time_percentage != tmp->old_time_percentage ||
 #endif
 				    tmp->prio != new_tmp->prio
--- origsrc/xfce4-taskmanager-0.4.1/src/taskmanager-linux.c	2008-09-15 16:17:08.000000000 -0500
+++ src/xfce4-taskmanager-0.4.1/src/taskmanager-linux.c	2009-02-11 23:07:09.015625000 -0600
@@ -140,7 +140,9 @@ static struct task get_task_details(gint
 
 	if (pagesize == 0)
 	  {
+#ifndef __CYGWIN__
 		pagesize = sysconf(_SC_PAGESIZE);
+#endif
 		if (pagesize == 0)
 			pagesize = 4*1024;
 	  }
@@ -205,6 +207,9 @@ static struct task get_task_details(gint
 
 	task.old_time = task.time;
 	task.time = stime + utime;
+#ifdef __CYGWIN__
+	task.time /= 10; // but why ???
+#endif
 	task.time_percentage = 0;
 	task.rss *= pagesize;
 
